lisa/testselector.py

```diff
@@ -139,13 +139,14 @@ def _force_check(
     return is_skip
 
 
-def _apply_filter(
+def _apply_filter(  # noqa: C901
```

Comment:
What's this?

Reply:
It's the flake8 way of disabling a warning for a particular line of code. In this case, C901 is the complexity warning that was enabled. I'd like the setting turned on so we can be see the warning on new code, but don't feel that we need to correct the existing violations. (Also, "noqa" probably comes from "no quality assurance.")